benchmarks/benchmark-euphoria-common/build.gradle

```diff
@@ -0,0 +1,7 @@
+dependencies {
+  compile project(':benchmarks:benchmark-model')
+  compile project(':euphoria-core')
+  compile project(':euphoria-hadoop')
+  compile project(':euphoria-kafka')
+  compile "com.typesafe:config:${configVersion}"
```

Comment:
Maybe findbugs should be in all dependencies ... Or could it be made inherited from some sort of parent?

Reply:
findbugs are applied to all subprojects in root build.gradle (dependency is need only in case you need to add suppressing annotations)